crates/bevy_ecs/src/system/system.rs

```diff
@@ -17,6 +17,15 @@ impl SystemId {
 }
 
 /// An ECS system that can be added to a [Schedule](crate::schedule::Schedule)
+///
+/// Systems are functions with all arguments implementing [SystemParam].
+///
+/// Systems are added to an application using `AppBuilder::add_system(my_system.system())`
+/// or similar methods, and will run once per pass of the main loop.
```

Comment:
```suggestion
/// or similar methods, and will generally run once per pass of the main loop.
```
I was trying to find balance between brevity and detailing out the exceptions and instead failed to bring them up at all 🤦 
I don't have a better solution.

Reply:
I agree: this is a good compromise in these docs for now.